can't `make binary-cross` at HEAD

``` console
Tunas-MacBook-Pro:kompose tuna$ make binary-cross
CGO_ENABLED=1 ./script/make.sh binary-cross
---> Making bundle: binary-cross (in .)
Number of parallel builds: 3

-->      darwin/386: command-line-arguments
-->       linux/386: command-line-arguments
-->     linux/amd64: command-line-arguments
-->   windows/amd64: command-line-arguments
-->     windows/386: command-line-arguments
-->    darwin/amd64: command-line-arguments

6 errors occurred:
--> linux/amd64 error: exit status 1
Stderr: can't load package: package command-line-arguments: cannot find package "command-line-arguments" in any of:
    /usr/local/go/src/command-line-arguments (from $GOROOT)
    /Users/tuna/Workspace/gocode/kompose/src/command-line-arguments (from $GOPATH)

--> darwin/386 error: exit status 1
Stderr: can't load package: package command-line-arguments: cannot find package "command-line-arguments" in any of:
    /usr/local/go/src/command-line-arguments (from $GOROOT)
    /Users/tuna/Workspace/gocode/kompose/src/command-line-arguments (from $GOPATH)

--> linux/386 error: exit status 1
Stderr: can't load package: package command-line-arguments: cannot find package "command-line-arguments" in any of:
    /usr/local/go/src/command-line-arguments (from $GOROOT)
    /Users/tuna/Workspace/gocode/kompose/src/command-line-arguments (from $GOPATH)

--> windows/386 error: exit status 1
Stderr: can't load package: package command-line-arguments: cannot find package "command-line-arguments" in any of:
    /usr/local/go/src/command-line-arguments (from $GOROOT)
    /Users/tuna/Workspace/gocode/kompose/src/command-line-arguments (from $GOPATH)

--> darwin/amd64 error: exit status 1
Stderr: can't load package: package command-line-arguments: cannot find package "command-line-arguments" in any of:
    /usr/local/go/src/command-line-arguments (from $GOROOT)
    /Users/tuna/Workspace/gocode/kompose/src/command-line-arguments (from $GOPATH)

--> windows/amd64 error: exit status 1
Stderr: can't load package: package command-line-arguments: cannot find package "command-line-arguments" in any of:
    /usr/local/go/src/command-line-arguments (from $GOROOT)
    /Users/tuna/Workspace/gocode/kompose/src/command-line-arguments (from $GOPATH)

make: *** [binary-cross] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -18,7 +18,8 @@ binary:
 	CGO_ENABLED=1 ./script/make.sh binary
 
 binary-cross:
-	CGO_ENABLED=1 ./script/make.sh binary-cross
+	# CGO_ENABLED=1 ./script/make.sh binary-cross
+	./script/make.sh binary-cross
 
 clean:
 	./script/make.sh clean

--- a/script/binary-cross
+++ b/script/binary-cross
@@ -24,4 +24,4 @@ rm -f kompose*
 gox "${OS_PLATFORM_ARG[@]}" "${OS_ARCH_ARG[@]}" \
     -output="bundles/kompose_{{.OS}}-{{.Arch}}/kompose" \
     "${BUILD_FLAGS[@]}" \
-    main.go
+    .

